core/src/main/java/com/yahoo/imapnio/async/request/EnableCommand.java

```diff
@@ -0,0 +1,74 @@
+package com.yahoo.imapnio.async.request;
+
+import java.nio.charset.StandardCharsets;
+
+import javax.annotation.Nonnull;
+
+import com.yahoo.imapnio.async.exception.ImapAsyncClientException;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+
+/**
+ * This class defines imap enable command request from client. RFC5161 ABNF: https://tools.ietf.org/html/rfc5161
+ *
+ * <pre>
+ * capability =/ "ENABLE"
+ *
+ * command-any =/ "ENABLE" 1*(SP capability)
+ *
+ * response-data =/ "*" SP enable-data CRLF
+ *
+ * enable-data = "ENABLED" *(SP capability)
+ * </pre>
+ *
+ */
+public class EnableCommand extends ImapRequestAdapter {
+
+    /**
+     * Initializes a @{code EnableCommand}.
+     *
+     * @param capabilities List of capability to enable
+     */
+    public EnableCommand(@Nonnull final String[] capabilities) {
+        this.capabilities = capabilities;
+    }
+
+    @Override
+    public void cleanup() {
+        capabilities = null;
+    }
+
+    @Override
+    public ByteBuf getCommandLineBytes() throws ImapAsyncClientException {
+        final ByteBuf sb = Unpooled.buffer(ENABLE_BUF_LEN);
+
+        sb.writeBytes(ENABLE_SP_B);
+
+        for (int i = 0, len = capabilities.length; i < len; i++) {
+            sb.writeBytes(capabilities[i].getBytes(StandardCharsets.US_ASCII));
```

Comment:
have a comment as following
// capability ABNF is:
// capability      = ("AUTH=" auth-type) / atom